docs/references/architecture/recipes/bridge.md

```diff
@@ -79,7 +80,9 @@ graph TD
   style D fill:#ccc,stroke:#333,stroke-width:4px,color:white
```

Comment:
One thing I learned during a Mermaid dependency update is that we can write a custom theme and can pull out any of these Style line you see here into that theme. I think this would also allow these charts to be rendered in the GitHub UI similar to the one I linked above. 

Not at all concerned about that for this PR, but if that's something you'd be interested in helping out with then feel free to give a shout!

Reply:
Actually, I've just found out that it's these `style` lines that break Docusaurus rendering.
With them, the  ` ```mermaid` syntax works on GitHub but not with Docusaurus.
Without them, both work (but with no styling). I'll look at those themes.

Reply:
(sorry, I did not see your other comment before !)